build.sbt

```diff
@@ -655,6 +655,7 @@ lazy val flinkDevModel = (project in flink("management/dev-model"))
   .dependsOn(
     extensionsApi,
     commonComponents,
+    flinkBaseUnboundedComponents,
```

Comment:
This dependency is needed to use Event Generator in designer tests

Reply:
It is not a good way to go, components modules are added dynamically, not statically, you should change the classpath inside the config to make them work instead of adding dependency

Reply:
Fixed, without dependency